tools/run_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+ # run setup script
+node ./tools/loadTestFiles.js && 
+
+# if no arguments run both tests
+if [ $# -eq 0 ]; then
+
+    # run node tests
+    mocha ./test/node
```

Comment:
Can you execute this via `npx`?

Reply:
the original script used "mocha ./test/node"
Node modules doesn't have a mongodb-test-runner in js-bson. I could do "npx mocha ./test/node", but I don't know if that's what you're looking for.

Reply:
Yes, I'd like to see all of the scripts that are installed with `npm` use `npx` to invoke them.

Reply:
I updated the node and karma tests to use npx, but now the node 4 and 6 tests fail in travis because they don't have npx.